internal/provider/resources/install/aws/staged.go

```diff
@@ -188,6 +193,22 @@ func (r *StagedAws) put(ctx context.Context, diags *diag.Diagnostics, plan *tfsd
 }
 
 func (r *StagedAws) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	diags := &resp.Diagnostics
+	diags.Append(req.Plan.Get(ctx, &stagedAwsModel{})...)
+	if diags.HasError() {
+		return
+	}
+
+	throwaway_response := struct{}{}
+	err := r.data.Post(r.itemBasePath(), struct{}{}, &throwaway_response)
+	if err != nil {
```

Comment:
I had to add this because I was getting the following error: 
<img width="406" alt="image" src="https://github.com/p0-security/terraform-provider-p0/assets/12995427/0b4ff255-8e28-4a5e-8c93-159802b472fa">

This is the same as the `Upsert` code in the install resources. I didn't use `installresources.Install` because the aws/staged instance does not implement any of the shared methods: `GetId func(data any) *string, GetItemJson, FromJson, ToJson`

cc: @nbrahms